### PR TITLE
HW1_pd2654 Part 2 and 3

### PR DESCRIPTION
**Pull request by MirrorCraze**
Add seven_wonders.md (Proposal) and numpy.md (Contributions)

Edit pd2654.md

### CODE REVIEW

**Review by timkpaine — CHANGES_REQUESTED** [review]
remove `contribution.md` and `proposal.md`

**Comment by MirrorCraze** [comment]
> remove `contribution.md` and `proposal.md`

contribution.md and proposal.md is removed.

File: pd2654.md [hunk 1]
```diff
@@ -6,17 +6,11 @@ UNI: pd2654
...
-to auto-save models.
-4. Cleaning my own project of 7 Wonders board game. I wrote a 7 Wonders Board Game entirely in Python for testing the reinforcement learning algorithm
-in OpenAI Gym. However, I would like to extend my code so that other people can also test their own model.
+## Project Proposal
+[Proposal](https://github.com/MirrorCraze/project-proposals-s2023/blob/HW1/projects/python/seven_wonders.md)
 
 ## Project Contributions
-[Numpy](https://github.com/numpy/numpy)
-[RLCard](https://github.com/datamllab/rlcard)
+[Contribution](https://github.com/MirrorCraze/project-proposals-s2023/blob/HW1/projects/python/numpy.md)
```
Comment (timkpaine) [inline pd2654.md:16]: use relative path

**Comment by timkpaine** [comment]
lgtm

**Pull request closed without merging**